### DISCUSSION TITLE
NullReferenceException on ServiceStack.ServiceStackHost.Release(Object instance)

### DISCUSSION THREAD

**Discussion by lucas-rvezy** [ServiceStack/Discuss#96] [Q&A]
We are getting the exception below that do not give us much information on what is causing it.

```
System.NullReferenceException: Object reference not set to an instance of an object.
   at ServiceStack.ServiceStackHost.Release(Object instance) in /home/runner/work/ServiceStack/ServiceStack/ServiceStack/src/ServiceStack/ServiceStackHost.cs:line 1408
```

It is just ignoring all the request logs we added and we couldn't find the cause of it. We couldn't replicate it locally. It only throw this exception on production.

**Comment by mythz** [comment]
The exception says the NRE happens here:

https://github.com/ServiceStack/ServiceStack/blob/main/ServiceStack/src/ServiceStack/ServiceStackHost.cs#L1408

The only reason why `Container` would be null would be after the AppHost.Dispose() is called when the App is shutting down.
